src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs

```diff
@@ -119,6 +119,7 @@ public NamespaceManager(IList<string> addresses, TokenProvider tokenProvider)
         ///                                      See <see cref="NamespaceManagerSettings.TokenProvider "/> to know more about the supported types.</exception>
         public NamespaceManager(Uri address, TokenProvider tokenProvider)
         {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
Is there a reason we are setting this explicitly?  Check the [TLS Guidelines for .NET](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls) for more information on what we should be doing, which is to not hardcode TLS 1.2 if we can help it

Reply:
removed.

Reply:
it was locally issue